runtime/src/lib.rs

```diff
@@ -500,6 +511,50 @@ impl pallet_collator_selection::Config for Runtime {
     type WeightInfo = ();
 }
 
+parameter_types! {
+	pub MaximumSchedulerWeight: Weight = Perbill::from_percent(60) *
```

Comment:
I see[ moonbeam ](https://github.com/moonbeam-foundation/moonbeam/blob/master/runtime/moonbeam/src/lib.rs#L202)has 75%, curious where you inspired this from

Reply:
EVM is heavy? :p

Reply:
I set a very conservative restriction as a safe start.